src/store/sagas/wallets/index.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import { walletsRootSaga as walletsSagas } from './core'
+import * as walletsAddressesSagas from './addresses'
+import * as walletsBackupSagas from './backup'
+import * as walletsCreateSagas from './create'
+import * as walletsDeleteSagas from './delete'
+import * as walletsImportSagas from './import'
+import * as walletsRenameSagas from './rename'
+import * as walletsRenameAddressSagas from './renameAddress'
+import * as walletsStarSagas from './start'
```

Comment:
The same point about imports
Why we need to change existed notation - https://github.com/jibrelnetwork/jwallet-web/blob/master/src/routes/Wallets/sagas/index.js?

What is wrong with your version:
you do `import * as walletsCreateSagas from './create'`
but we need just one rootSaga in this file.

Let's discuss it offline